README.md

```diff
@@ -1,3 +1,6 @@
+![Linux Post Commit Checks](https://github.com/intel/llvm/workflows/Linux%20Post%20Commit%20Checks/badge.svg)
```

Comment:
What about putting links to actual actions jobs?

For example, in the translator we have:
```
[![Build Status](https://travis-ci.org/KhronosGroup/SPIRV-LLVM-Translator.svg?branch=master)](https://travis-ci.org/KhronosGroup/SPIRV-LLVM-Translator)
```